packages/core/src/interpreter.ts

```diff
@@ -140,7 +143,7 @@ export class Interpreter<
       clearTimeout: (id) => {
         return clearTimeout(id);
       }
-    },
+    } as Clock,
```

Comment:
For some reason, `tsc` has started to infer the `setTimeout`'s return type of this default clock to be of type `NodeJS.Timeout`. That in turn has caused it to include `/// <reference types="node" />` at the top of the file and that has created the reported issue.